### PR TITLE
Snackbar over bottom toolbar fix

### LINKED ISSUE #579: Snackbar appears over bottom toolbar
### Are you ready to report?
   - Your issue may already be reported! Please search on the [issue track](../) before creating one.
   - Ensure you have tested with the last (dev) version of the software.  (If not, and if possible, run your test again with the most recent version available.)

### Environment
  * version of the software you use : Android Studio 3.0.1
  * device / operating system : Nexus 5X / Android 8.1.0
  
### The Bug:

  * What do you obtain? A copy of the error message or a screenshot is always useful.

The snackbar appear over bottom navigation bar when we bookmark / remove bookmark any item.

  * When does this occur? 

#### Steps to reproduce:

  1. Go to setting and enable bottom toolbar.
  2. Now an the main page try bookmarking any article
  3. Try removing the bookmark also.
  ...
  
  * What should be the behaviour from your point of view? How do you expect the service to work?
As per the [Material UI guidelines](https://material.io/guidelines/components/bottom-navigation.html#bottom-navigation-specs) the snackbar should appear above the bottom navigation bar as shown in the screenshot below.

### Other Comments:
  Would like to work on it.

### PR DESCRIPTION
**Pull request by panvica10**
Fixes #579 

Changes: Created PageBottomTabLayoutSnackBarBehavior and binded to PageBottomTabLayout in main.xml, not used Relative Layouts removed

Screenshots/GIF for the change: screenshot_2018-10-07-04-00-01 
screenshot_2018-10-07-04-00-19

### CODE REVIEW

**Comment by panvica10** [comment]
@kiwix/lieutenants

File: app/src/main/java/org/kiwix/kiwixmobile/views/PageBottomTabLayoutSnackBarBehavior.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import android.support.design.widget.CoordinatorLayout.Behavior;
+import android.support.design.widget.Snackbar;
+import android.support.v4.view.ViewCompat;
+import android.util.AttributeSet;
+import android.view.Gravity;
+import android.view.View;
+
+public class PageBottomTabLayoutSnackBarBehavior extends Behavior<View> {
+
+    public PageBottomTabLayoutSnackBarBehavior() {
```
Comment (mhutti1) [inline app/src/main/java/org/kiwix/kiwixmobile/views/PageBottomTabLayoutSnackBarBehavior.java]: Please use 2 space indents. Otherwise this looks good.

Comment (panvica10) [reply app/src/main/java/org/kiwix/kiwixmobile/views/PageBottomTabLayoutSnackBarBehavior.java]: Fixed :)

**Comment by abdulwd** [comment]
@panvica10 Can you rebase your branch on master?

**Pull request closed without merging**